frontend/cypress/e2e/complaint-search.v2.cy.ts

```diff
@@ -147,4 +147,47 @@ describe("Complaint Search Functionality", () => {
         expect(length).to.eq(5);
       });
   });
+
+  it.only("Can retains search parameters when navigating back to the search page", () => {
+    cy.visit("/");
+    cy.waitForSpinner();
+
+    //-- load the ERS conflicts and set filters
+    cy.navigateToTab(complaintTypes[1], false);
+    cy.get("#comp-zone-filter").click({ force: true });
+    cy.waitForSpinner();
+    cy.get("#comp-zone-filter").should("not.exist");
+
+    cy.get("#comp-filter-btn").click({ force: true });
+    cy.selectItemById("zone-select-id", "South Island");
+    cy.get("#comp-zone-filter").should("exist");
+
+    cy.get("#complaint-search").click({ force: true });
+    cy.get("#complaint-search").clear().type("wildlife{enter}"); //-- {enter} will perform an enter keypress
+
+    cy.get("#map_toggle_id").click({ force: true });
+    cy.verifyMapMarkerExists(true);
+
+    cy.get("#multi-point-map")
+      .find("div.leaflet-marker-icon")
+      .should(({ length }) => {
+        expect(length).to.eq(2);
```

Comment:
this is failing as I think there is only result that would show on the map.

Reply:
It should be fixed now, I was using local instead of Cypress directly to pick the data, that's my bad